### PR TITLE
[OS-1091] - Adding a systemd service for kano-sync

### PR DESCRIPTION
**Pull request by skarbat**
We are moving the kano-sync to be run as a systemd service, in the context of the interactively logged in user. This PR provides this service. Tested on the RPi.

Note that currently the service fails due to issues contacting the network services.

### CODE REVIEW

File: systemd/user/kano-sync.service [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# the internet-available-user.target
+#
+
+[Unit]
+Description=Kano Sync
+Before=internet-available-user.target
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/kano-sync --upload-tracking-data --silent
```
Comment (radujipa) [inline systemd/user/kano-sync.service]: Also, should we consider sync-ing everything in this service? User data, tracking data, etc.?

**Comment by skarbat** [comment]
I believe that the DHCP hook will trigger the `internet-available` targets and therefore `kano-sync` will wake up? Still unsure on your implementation here @radujipa . I have tested to start the service manually and it works fine.

File: systemd/user/kano-sync.service (hunk 1 above)
Comment (skarbat) [reply systemd/user/kano-sync.service]: I would propose to fix one thing at a time. Right now this command fails due to - presumably - some mismatch in the way the new KW API expects this data, but I can't tell for sure.

**Comment by radujipa** [comment]
systemd will need to know to invoke your `kano-sync.service` automatically when `internet-available-user.target` is started. By default, systemd only starts services that are enabled and simply copying them on the system does not enable them.

Targets are a different story, these do not need to be enabled.

File: debian/kano-profile.postinst [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# Copyright (C) 2019 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPLv2
+#
+# Post debian package install script.
+#
+
+case "$1" in
+    configure)
+	# Enable user systemd services for all users.
+        systemctl --global enable kano-sync.service
```
Comment (radujipa) [inline debian/kano-profile.postinst]:
```suggestion
      systemctl --global enable kano-sync.service
```

**Comment by skarbat** [comment]
Thanks for the `prerm` changes @radujipa ! Merging in.

**Pull request merged**